prerequisites.py

```diff
@@ -148,6 +148,9 @@ def assert_not_root():
     if needed.
 
     """
+    if "--as-root" in sys.argv:
```

Comment:
Sorry for not being very precise - I meant the function documentation. Feel free to leave what you added though, I think it's nice to have it in the usage.